hols/oci-basic-setup/README.md

```diff
@@ -0,0 +1,276 @@
+   * `instance_*` - Use these parameters to specify the size and shape of the OCI Compute instance that you would like to be provisioned. Currently defaults to an available shape from an [Always Free Resources of an OCI Free Tier Tenancy](https://docs.oracle.com/en-us/iaas/Content/FreeTier/freetier_topic-Always_Free_Resources.htm).
+     ```terraform
+     instance_shape                      = "VM.Standard.E2.1.Micro"
+     instance_ocpus                      = 1
+     instance_shape_config_memory_in_gbs = 1
+     instance_os                         = "Oracle Linux"
+     instance_os_version                 = "8"
+     ```
+4. Use Terraform to execute the scripts to provision the OCI resources.
+   ```bash
+   terraform init
+   terraform plan
+   terraform apply -auto-approve
+   ```
+    
+### Prepare the Helidon OCI Application
+1. Go to the home directory from a Cloud Shell console:
+   ```shell
+   cd ~
+   ```
+2. Make sure that JDK 21 exists.
+   1. Run the following command to validate:
+      ```shell
+      java -version
+      ```
+   2. If it doesn't exist, download and install the appropriate binaries relevant to your OS platform. Below is an example of downloading and extracting a Linux Arm64 JDK compressed archive that will work with the default platform of Cloud Shell.
+      ```shell
+      curl -O https://download.oracle.com/java/21/latest/jdk-21_linux-aarch64_bin.tar.gz
+      tar -xvzf jdk-21_linux-aarch64_bin.tar.gz
+      ```
+   3. Set the JAVA_HOME and PATH environment variables to include the corresponding directories from the extracted JDK binaries. 
+      ```shell
+      export JAVA_HOME=~/jdk-21.0.7/
+      export PATH=$JAVA_HOME/bin/:$PATH
+      ```
+      **Note:** If you want the above environment variable settings to persist between Cloud Shell sessions, add the definitions to your `~/.bashrc` file.
+3. Make sure that Maven 3.8 or higher exists.
+    1. Run the following command to validate:
+       ```shell
+       mvn --version
+       ```
+    2. If it doesn't exist, download and install the binary archive of your choice from the [Apache Maven Download page](https://maven.apache.org/download.cgi). Below is an example of downloading and extracting a  Maven 3.9.9 binary tar.gz archive.
+       ```shell
+       curl -O https://dlcdn.apache.org/maven/maven-3/3.9.9/binaries/apache-maven-3.9.9-bin.tar.gz
+       tar -xvzf apache-maven-3.9.9-bin.tar.gz
+       ```
+    3. Add the extracted mvn bin subdirectory into the PATH environment variable
+       ```shell
+       export PATH=~/apache-maven-3.9.9/bin/:$PATH
+       ```
+       **Note:** If you want the above environment variable setting to persist between Cloud Shell sessions, add the definition to your `~/.bashrc` file.
+4. Install the Helidon CLI generic distribution.
+    1. Download and extract the Helidon CLI generic distribution
+       ```shell
+       curl -L -O https://github.com/helidon-io/helidon-build-tools/releases/download/3.0.6/helidon-cli.zip
```

Comment:
The recommended approach to download CLI is `curl -L -O https://helidon.io/cli/latest/linux/helidon` based on https://helidon.io/docs/v4/about/cli

Reply:
Unfortunately, that will only work for Linux x64 platform. Cloud Shell no runs on Linux ARM by default and we don't have a distro for it yet, so the generic java distro is the only option for now.